crates/ruff/src/rules/pydocstyle/settings.rs

```diff
@@ -97,19 +97,35 @@ pub struct Options {
     /// specified decorators. Unlike the `pydocstyle`, Ruff accepts an array
     /// of fully-qualified module identifiers, instead of a regular expression.
     pub ignore_decorators: Option<Vec<String>>,
+    #[option(
+        default = r#"[]"#,
+        value_type = "list[str]",
+        example = r#"
+            property-decorators = ["gi.repository.GObject.Property"]
+        "#
+    )]
+    /// Consider any method decorated with one of these decorators as a property,
+    /// and consequently allow a docstring which is not in imperative mood.
+    /// Unlike pydocstyle, supplying this option doesn't disable standard
+    /// property decorators - `@property` and `@cached_property`.
```

Comment:
I think this behavior makes much more sense, but I can make it match pydocstyle. What do you think?

Reply:
I prefer what you have here.

Reply:
Oh, although I guess `staticmethod_decorators` and `classmethod_decorators` don't work that way. I feel like they _should_ though. Fine to leave it for now, maybe we change those other settings semantics in a follow-up.